lib/r10k/forge/module_release.rb

```diff
@@ -48,10 +44,6 @@ class ModuleRelease
       #   @return [Pathname] Where the module will be unpacked to.
       attr_accessor :unpack_path
 
-      # @!attribute [rw] unpack_root
-      #   @return [Pathname] Directory where the module will be unpacked to.
-      attr_accessor :unpack_root
-
       # @param full_name [String] The hyphen separated name of the module
       # @param version [String] The version of the module
       def initialize(full_name, version)
```

Comment:
No longer need to set `download_root` and `unpack_root` in the initializer.

Reply:
Thanks, updated.